android/java/org/chromium/chrome/browser/crypto_wallet/activities/AssetDetailActivity.java

```diff
@@ -8,7 +8,10 @@
 import android.annotation.SuppressLint;
 import android.app.Activity;
 import android.content.Intent;
+import android.graphics.Bitmap;
```

Comment:
I couldn't find where these are used:
`import android.graphics.Bitmap;`
`import org.chromium.brave_wallet.mojom.AssetPrice;`
`import org.chromium.brave_wallet.mojom.TransactionStatus;`
`import org.chromium.brave_wallet.mojom.TransactionType;`
`import java.text.DateFormat;`
`import java.text.SimpleDateFormat;`
`import java.util.Arrays;`
`import java.util.Date;`
`import java.util.HashMap;`
`import java.util.Locale;`

Reply:
kk, my bad, I first had all the logic in that file, but moved to Utils after